src/patternfly/components/DataList/data-list.scss

```diff
@@ -368,6 +404,15 @@
   &.pf-m-flex-5 { flex-grow: 5; }
 }
 
+// Data list text
+.pf-c-data-list__text {
+  max-width: var(--pf-c-data-list__cell--MaxWidth);
+  overflow: var(--pf-c-data-list__cell--Overflow);
+  text-overflow: var(--pf-c-data-list__cell--TextOverflow);
+  word-break: var(--pf-c-data-list__cell--WordBreak);
+  white-space: var(--pf-c-data-list__cell--WhiteSpace);
+}
```

Comment:
I'd probably pair `.pf-c-data-list__cell` and `.pf-c-data-list__text` and apply the shared properties.